doc/modules/neural_networks.rst

```diff
@@ -1,11 +1,140 @@
 .. _neural_network:
 
+==================================
+Neural network models (supervised)
+==================================
+
+.. currentmodule:: sklearn.neural_network
+
+Randomly weighted neural networks
```

Comment:
Maybe remove this whole section, I saw you have another one below for random basis functions?